packages/web3auth/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/web3auth",
-  "version": "2.3.1",
+  "version": "2.4.0-alpha.1",
```

Comment:
Thanks @bengobeil  - We will also want to add this new version update to `packages/demo/package.json` and run `yarn && yarn check-all` from the root of the project and ensure there are not any errors